lib/v2/screens/receive/components/receive_selection_card.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+
+class ReceiveSelectionCard extends StatelessWidget {
+  final String title;
+  final Widget? icon;
```

Comment:
make this not nullable

Reply:
Inkwell widget's onTap is nullable, is really necessary make it no nullable?
Also @gguij004 if you could rename it **onTap** because **callback** is more ambiguous in the sense of when it will run.

Reply:
you need on tap to make this widget work well. 
he made it required in the constructor already. 
Might as well make it not null

Reply:
Fix, thank you both.